ports/espressif/boards/sensebox_eye_esp32s3/mpconfigboard.mk

```diff
@@ -0,0 +1,21 @@
+USB_VID = 0x303A
+USB_PID = 0x82D2
+
+USB_PRODUCT = "senseBox-eye ESP32S3"
+USB_MANUFACTURER = "Espressif"
```

Comment:
```suggestion
USB_MANUFACTURER = "senseBox"
```

Reply:
@tannewt thank you! Changed it for both senseBox Boards